### PR TITLE
fallback to current folder when caller stack is empty on import model

### PR DESCRIPTION
**Pull request by antt001**
### Pull Request check-list

_Please make sure to review and check all of these items:_

- [x] Does `npm run test` or `npm run test-DIALECT` pass with this change (including linting)?
- [ ] Does the description below contain a link to an existing issue (Closes #[issue]) or a description of the issue you are solving?
- [ ] Have you added new tests to prevent regressions?
- [ ] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?
- [ ] Did you update the typescript typings accordingly (if applicable)?
- [ ] Did you follow the commit message conventions explained in [CONTRIBUTING.md](https://github.com/sequelize/sequelize/blob/master/CONTRIBUTING.md)?

### Description of change

When importing the model, if the caller stack is empty the import will fail with an error stating that path parameter is not a string. this patch adds fallback to current dir to avoid the error and continue the import process to the callback function(e.g. second parameter)

### CODE REVIEW

**Comment by sushantdhiman** [comment]
Can you give me an example what triggers this error?

**Comment by antt001** [comment]
> Can you give me an example what triggers this error?
Sure. It was happening to me. It is when you use sequelize.import('some_string', (sequelize, DataTypes) => { return sequelize.define({...}); } , while inside package build with webpack.
let me rephrase it,
I have a module built with webpack, that imports the models.
I require this module from few other modules.
So if I require it from a module that is also compiled with webpack, then it causes the bug.
If I require it from a regular node.js app (not bundled with webpack) it doesn't cause the bug.

**Comment by antt001** [comment]
the more declarative way to fix it might be to surround the call to  `path.dirname` with try - catch, but this way is shorter, and it does the job.

**Comment by sushantdhiman** [comment]
Thanks, With v6 we are going to remove `import` https://github.com/sequelize/sequelize/issues/11739.
